public/app/core/components/PermissionList/AddPermission.tsx

```diff
@@ -64,7 +64,7 @@ class AddPermissions extends Component<Props, NewDashboardAclItem> {
   };
 
   onPermissionChanged = (permission: SelectableValue<PermissionLevel>) => {
-    this.setState({ permission: permission.value });
+    this.setState({ permission: permission.value ?? PermissionLevel.View });
```

Comment:
I would rather not assume much about what a default permission level should be. I think better would be just to do `permission!.value` have a comment and let it blow up if something changes upstream.